### PR TITLE
Create output directory if it does not exist

### PR DESCRIPTION
**Pull request by SemenchenkoVitaliy**
/cc @lundibundi @belochub @nechaido

### CODE REVIEW

File: bin/cli.js [hunk 1]
```diff
@@ -187,8 +188,14 @@ if (config.writeToStdout) {
   }
 } else {
   console.log('Writting output to following files...');
-  for (const [file, data] of output) {
-    fs.writeFileSync(file, data);
-    console.log(file);
-  }
+  common.mkdirp(config.outputDir, err => {
```
Comment (lundibundi) [inline bin/cli.js]: Will this work correctly for `config.outputDir === '.'`?

Comment (SemenchenkoVitaliy) [reply bin/cli.js]: Yes, why it shouldn't?

Comment (lundibundi) [reply bin/cli.js]: Ok then, just making sure.

**Comment by lundibundi** [comment]
Landed in https://github.com/metarhia/metadoc/commit/6394ae360d941be0c2ebe5ffc2a6a998b6709c46.

**Pull request closed without merging**